core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation-dropdown.directive.html

```diff
@@ -0,0 +1,151 @@
+        <i class="material-icons">&#xE037;</i>
+        <span  style="padding-left: 10px;">Preview</span>
+      </a>
+    </li>
+    <li ng-class="{'active': getActiveTabName() === 'settings'}">
+      <a href="#" uib-tooltip="Settings" tooltip-placement="bottom" ng-click="selectSettingsTab()"
+         class="protractor-test-settings-tab" style="padding-left: 4px;">
+        <i class="material-icons">&#xE8B8;</i>
+        <span  style="padding-left: 10px;">Settings</span>
+      </a>
+    </li>
+    <li ng-if="userIsLoggedIn" ng-class="{'active': getActiveTabName() === 'stats'}">
+      <a href="#" uib-tooltip="Statistics" tooltip-placement="bottom" ng-click="selectStatsTab()"
+         class="protractor-test-stats-tab" style="padding-left: 4px;">
+        <i class="material-icons">&#xE24B;</i>
+        <span style="padding-left: 10px;">Statistics</span>
+      </a>
+    </li>
+    <li ng-if="isImprovementsTabEnabled()" ng-class="{'active': getActiveTabName() === 'improvements'}" ng-click="selectImprovementsTab()">
+      <a href="#" uib-tooltip="Improvements" tooltip-placement="bottom"
+         class="protractor-test-improvements-tab" style="padding-left: 4px;">
+        <i class="material-icons">trending_up</i>
+        <span style="padding-left: 10px;">Improvements</span>
+      </a>
+    </li>
+    <li ng-class="{'active': getActiveTabName() === 'history'}">
+      <a href="#" uib-tooltip="History" tooltip-placement="bottom" ng-click="selectHistoryTab()"
+         disabled="ExplorationRightsService.isCloned()"
+         class="protractor-test-history-tab" style="padding-left: 4px;">
+        <i class="material-icons">&#xE192;</i>
+        <span style="padding-left: 10px;">History</span>
+      </a>
+    </li>
+    <li  ng-if="isFeedbackTabEnabled()" ng-class="{'active': getActiveTabName() === 'feedback'}" ng-click="selectFeedbackTab()">
+      <a href="#" uib-tooltip="Feedback" tooltip-placement="bottom"
+         class="protractor-test-feedback-tab" style="padding-left: 4px;">
+        <i class="material-icons">&#xE87F;</i>
+        <span style="padding-left: 10px;">Feedback</span>
+      </a>
+      <div ng-show="getOpenThreadsCount()" class="oppia-exploration-open-threads-indicator oppia-exploration-open-threads-color">
+        <span class="oppia-exploration-open-threads-count">
+            <[getOpenThreadsCount()]>&nbsp;open
+        </span>
+      </div>
+    </li>
+    <li ng-if="userIsLoggedIn" uib-dropdown class="uib-dropdown uib-popover ng-cloak" popover-is-open="popoverControlObject.postTutorialHelpPopoverIsShown"
+        popover-placement="bottom" popover-trigger="'none'"
+        uib-popover="To get help in the future, click here!">
+      <a href="#" uib-tooltip="Help" tooltip-placement="bottom"
+         ng-click="showUserHelpModal()" style="padding-left: 4px;">
+        <i class="material-icons">&#xE887;</i>
+        <span style="padding-left: 10px;">Help</span>
+      </a>
+    </li>
+  </ul>
+</div>
```

Comment:
Add a newline at EOF. If you do it correctly, you shouldn't see the red symbol in GitHub's files changed tab
![Screenshot from 2019-08-20 01-29-15](https://user-images.githubusercontent.com/10575562/63331159-f0861b80-c2e9-11e9-8be4-18c0e1e3bd5b.png)
:

Reply:
I have given a EOF.